test/acceptance/cli-test.js

```diff
@@ -319,40 +382,64 @@ describe('ember-template-lint executable', function() {
       });
 
       describe('given a directory with errors and a lintrc with rules', function() {
-        it('should print properly formatted error messages', function() {
-          let result = run(['.', '--config-path', '../with-errors/.template-lintrc'], {
-            cwd: './test/fixtures/without-errors',
+        it('should print properly formatted error messages', async function() {
+          setProjectConfigWithoutErrors();
+
+          let projectWithErrors = new Project();
```

Comment:
Don't we have a shared `project` instance from the top-most `beforeEach`?

Reply:
so for this particular case, the `--config-path` is using another project's `.template-lintrc`, and thats what I am trying to mimic here.

Reply:
I don’t really understand what you mean. This test is just saying “given a project with errors it prints errors and warnings correctly”, this test shouldn’t need to use configPath (it only used it before because it was inconvenient to chdir and back).